gradle-plugin/openshift/src/main/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftResourceTask.java

```diff
@@ -20,7 +20,7 @@
 import java.util.Properties;
 
 public class OpenShiftResourceTask extends KubernetesResourceTask {
-  private static final String DOCKER_IMAGE_USER = "docker.image.user";
+  private static final String DOCKER_IMAGE_USER = "jkube.image.user";
```

Comment:
This property (both for Mojo and Task is used later). We should expose that constant and use it here (and in Mojo) instead.